### DISCUSSION TITLE
各位同学在用pdf查重的时候又没有出现过参考文献识别失效的情况？都是怎么解决的？

### DISCUSSION THREAD

**Discussion by bakulaibuji** [sjtug/SJTUThesis#874] [提问] [answered]
各位同学在用pdf查重的时候又没有出现过参考文献识别失效的情况？都是怎么解决的？

**Comment by LogCreative** [comment, marked as answer]
有可能你对PDF做额外的操作了，一般不会出现这个情况，参见 https://shuiyuan.sjtu.edu.cn/t/topic/62769 。
